### PR TITLE
[BROS-287] Build Connect SDK Sample App

### PR DESCRIPTION
**Pull request by eepDev**
## Changes
- Add sample Chat and Support apps
- Update gradle, bump libraries version
- Initialize SDKs in Application class
- Set up Travis

### Reviewers
@zendesk/two-brothers @baz8080 @brendan-fahy @schlan 

### FYI
@KevinLambe

### References
- [Jira](https://zendesk.atlassian.net/browse/BROS-287)

### Risks
* Low

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    - android-sdk-license-[0-9a-f]{8}
+
+branches:
+  only:
+    - master
+
+script:
+  - ./scripts/shell/travis.sh
+
+after_script:
```
Comment (schlan) [inline .travis.yml]: we don't need that

File: chat_api_sample/build.gradle [hunk 2]
```diff
@@ -6,30 +6,25 @@ android {
...
         dataBinding {
             enabled true
         }
     }
-}
-
-repositories {
-    maven { url 'https://zendesk.artifactoryonline.com/zendesk/repo' }
+    lintOptions {
+        abortOnError false
```
Comment (schlan) [inline chat_api_sample/build.gradle]: `abortOnError` should always be `true`, can you please add a lint config if there are any errors

File: build.gradle [hunk 3]
```diff
@@ -1,23 +1,24 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 ext {
```
Comment (schlan) [inline build.gradle]: a couple of comments for this section:
 - we don't need `minProviderSdkVersion` please remove it
 - `targetSdkVersion` should be 25
 - `zendeskSdkVersion` should be 1.7.3.1
 - `zopimSdkVersion` should be 1.3.2.1

File: chat_sample/build.gradle [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+}
+
+dependencies {
+    compile group: 'com.zopim.android', name: 'sdk', version: '1.3.2.1'
```
Comment (schlan) [inline chat_sample/build.gradle]: please use `zopimSdkVersion`

File: support_request_list_with_add_ticket_sample/src/main/java/com/zendesk/sample/requestlist/Global.java [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+                new AnonymousIdentity.Builder()
+                        .withNameIdentifier("{optional name}")
+                        .withEmailIdentifier("{optional email}")
+                        .withExternalIdentifier("{optional external identifier}")
+                        .build()
+        );
+
+        // b). JWT (Must be initialized with your JWT identifier)
+//        ZendeskConfig.INSTANCE.setIdentity(new JwtIdentity("{JWT User Identifier}"));
+        ZendeskConfig.INSTANCE.setIdentity(new JwtIdentity("bcarroll@zendesk.com"));
```
Comment (schlan) [inline support_request_list_with_add_ticket_sample/src/main/java/com/zendesk/sample/requestlist/Global.java]: remove?

File: build.gradle (hunk 3 above)
Comment (eepDev) [inline build.gradle]: done!

**Pull request merged**